setup.py

```diff
@@ -29,7 +29,7 @@
     author_email='data-analytics-team@palmetto.com',
     url='https://github.com/palmetto/palm-dbt',
     packages=find_namespace_packages(include=['palm', 'palm.*']),
-    package_data={'': ['palm/plugins/dbt/**/*']},
+    package_data={'': ['*.md', '*.sql', '*.yaml', '*.yml', '*.txt']},
```

Comment:
Why change back?

Reply:
It broked. for whatever reason package data needs this set explicitly or it won't resolve